tools/Azure.Mcp.Tools.ConfidentialLedger/src/ConfidentialLedgerSetup.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Azure.Mcp.Core.Areas;
+using Azure.Mcp.Core.Commands;
+using Azure.Mcp.Tools.ConfidentialLedger.Commands.Entries;
+using Azure.Mcp.Tools.ConfidentialLedger.Services;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Azure.Mcp.Tools.ConfidentialLedger;
+
+public class ConfidentialLedgerSetup : IAreaSetup
+{
+    public string Name => "confidentialledger";
+
+    public void ConfigureServices(IServiceCollection services)
+    {
+        services.AddSingleton<IConfidentialLedgerService, ConfidentialLedgerService>();
+        services.AddSingleton<LedgerEntryAppendCommand>();
+    }
+
+    public CommandGroup RegisterCommands(IServiceProvider serviceProvider)
+    {
+        var root = new CommandGroup(Name,
+            "Azure Confidential Ledger operations - Commands for appending and querying tamper-proof ledger entries backed by TEEs and blockchain-style integrity guarantees. Use these commands to write immutable audit records.");
```

Comment:
@anannya03 PTAL since you've been tuning tool descriptions a ton the past 3 weeks